### PR TITLE
PMM-3061 (subtask PMM-3069) describing agentless mysql and posgres…

### PR DESCRIPTION
**Pull request by fiowro**
…support

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-236/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-236/

File: doc/source/amazon-rds.rst [hunk 1]
```diff
@@ -1,5 +1,50 @@
...
+
+.. figure:: .res/graphics/png/metrics-monitor.add-rds-or-remote-instance.png
+
+.. _pmm.amazon-rds-agentless:
+
+Adding Remote DB instance to |pmm|
+********************************************************************************
+
+This is a quick method for users to add DBaaS instances without having to hook
+into the Cloud Provider's API, and with no need to have pmm-admin present on the
```
Comment (michaelcoburn) [inline doc/source/amazon-rds.rst]:
```suggestion
into the Cloud Provider's API, and with no need to have pmm-client installed or any exporters running on the
```

File: doc/source/amazon-rds.rst [hunk 2]
```diff
@@ -1,5 +1,50 @@
...
+monitored node. The drawback of this approach is having no node-specific
```
Comment (michaelcoburn) [inline doc/source/amazon-rds.rst]:
```suggestion
monitored node. The drawback of this approach is that you will not have visibility of host-level metrics (CPU, memory, and disk activity will not be captured nor displayed in PMM)
```

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-236/

File: doc/source/remote-instance.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+memory, and disk activity will not be captured nor displayed in |pmm|). 
+
+.. note: There is an alternative and more complex approach available for
+         |mysql-server|, which involves API-aware addition of an Amazon RDS /
+         Aurora DB instance.
+
+Both methods can be accessed in the |metrics-monitor| navigation menu by
+selecting the |pmm-add-instance| item in a |gui.pmm-dropdown| group:
+
+.. figure:: .res/graphics/png/metrics-monitor.cloud-menu.png
```
Comment (michaelcoburn) [inline doc/source/remote-instance.rst]: update this image to be from the PMM menu

File: doc/source/remote-instance.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+Two database servers are currently supported by this method: |postgresql| and
+|mysql|.
+
+.. figure:: .res/graphics/png/metrics-monitor.add-rds-or-remote-instance.png
```
Comment (michaelcoburn) [inline doc/source/remote-instance.rst]: scale down all images by 50%

File: doc/source/remote-instance.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+
+Adding a Remote PostgreSQL Instance
+================================================================================
+
+To add a remote PostgreSQL DB instance, user should fill in at least the
+hostname, username and password field, and optionally also non-default hostname
+and port ones:
+
+.. figure:: .res/graphics/png/metrics-monitor.add-remote-postgres-instance.png
```
Comment (michaelcoburn) [inline doc/source/remote-instance.rst]: scale down all images by 50%

File: doc/source/remote-instance.rst [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+
+Adding a Remote MySQL Instance
+================================================================================
+
+To add a remote MySQL DB instance, user should fill in at least the
+hostname, username and password field, and optionally also non-default hostname
+and port ones:
+
+.. figure:: .res/graphics/png/metrics-monitor.add-remote-mysql-instance.png
```
Comment (michaelcoburn) [inline doc/source/remote-instance.rst]: scale down all images by 50%

File: doc/source/remote-instance.rst [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+Viewing Remote MySQL and PostgreSQL Instances
+================================================================================
+
+After adding, both |amazon-rds| and remote instances can be seen in the RDS and
+Remote Instances` list: remote ones have `remote` keyword as a Region:  
+
+.. figure:: .res/graphics/png/metrics-monitor.add-rds-or-remote-instance1.png
```
Comment (michaelcoburn) [inline doc/source/remote-instance.rst]: scale down all images by 50%

File: doc/source/remote-instance.rst (hunk 3 above)
Comment (michaelcoburn) [reply doc/source/remote-instance.rst]: replace image with this one:
image
lets promote the two new screen updates by showing a highlight around them both, and around the pmm menu.

File: doc/source/remote-instance.rst [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+To add a remote MySQL DB instance, user should fill in at least the
+hostname, username and password field, and optionally also non-default hostname
+and port ones:
+
+.. figure:: .res/graphics/png/metrics-monitor.add-remote-mysql-instance.png
+
+Viewing Remote MySQL and PostgreSQL Instances
+================================================================================
+
```
Comment (michaelcoburn) [inline doc/source/remote-instance.rst]: Add a section that shows how to navigate to the Remote instances screen. Use this image:
image
show the highlight around PMM submenu, and around the RDS and Remote Instances link

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-236/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-236/

**Pull request merged**